cmd/report_test.go

```diff
@@ -0,0 +1,61 @@
+package cmd
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"go.k6.io/k6/execution"
+	"go.k6.io/k6/lib"
+	"go.k6.io/k6/lib/executor"
+	"go.k6.io/k6/lib/testutils"
+	"go.k6.io/k6/lib/types"
+	"gopkg.in/guregu/null.v3"
+)
+
+func TestCreateReport(t *testing.T) {
+	t.Parallel()
+	importedModules := []string{
+		"k6/http",
+		"my-custom-module",
+		"k6/experimental/webcrypto",
+		"file:custom-from-file-system",
+		"k6",
+		"k6/x/custom-extension",
+	}
+
+	outputs := []string{
+		"json",
+		"xk6-output-influxdb",
+	}
+
+	logger := testutils.NewLogger(t)
+	opts, err := executor.DeriveScenariosFromShortcuts(lib.Options{
+		VUs:      null.IntFrom(10),
+		Duration: types.NullDurationFrom(5 * time.Minute),
+	}, logger)
+	require.NoError(t, err)
+
+	s, err := execution.NewScheduler(&lib.TestRunState{
+		TestPreInitState: &lib.TestPreInitState{
+			Logger: logger,
+		},
+		Options: opts,
+	})
+	require.NoError(t, err)
+	s.GetState().ModInitializedVUsCount(6)
+	s.GetState().AddFullIterations(170)
+	s.GetState().MarkStarted()
+	time.Sleep(10 * time.Millisecond)
+	s.GetState().MarkEnded()
+
+	r := createReport(s, importedModules, outputs)
+	assert.NotEmpty(t, r.Version)
```

Comment:
Would make sense to switch to  (or add) something like?

```suggestion
	assert.Equal(t, consts.Version, r.Version)
```